packages/paste-core/primitives/tabs/stories/index.stories.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {Button} from '@twilio-paste/button';
+import {Paragraph} from '@twilio-paste/typography';
+import {Box} from '@twilio-paste/box';
+import {useUID} from 'react-uid';
+import {useTabsPrimitiveState, TabsPrimitive, TabsPrimitiveList, TabsPrimitivePanel} from '../src';
+
+const BaseExample: React.FC<{}> = () => {
+  const tabId = useUID();
+  const tab = useTabsPrimitiveState({selectedId: tabId});
+  return (
+    <>
+      <TabsPrimitiveList {...tab} aria-label="My tabs">
+        <Box marginRight="space20" display="inline-block">
```

Comment:
If you didn't want to use boxes you could just use a `Stack`

Reply:
True! Swappin'